.github/workflows/integration-tests-on-release.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions: ["v1.27.16", "v1.28.15", "v1.30.8", "v1.32.0"]
+        kubernetes-versions: ["v1.28.15", "v1.28.15", "v1.30.8", "v1.32.0"]
```

Comment:
Could we also update all the patch versions at the same time? For example, v1.32.3.

Reply:
done